junit5/src/test/java/org/jboss/weld/junit5/basic/SimpleTest.java

```diff
@@ -35,7 +38,8 @@
 
     @Test
     public void testFoo() {
-        Assertions.assertEquals("baz", weld.select(Foo.class).get().getBar());
+        assertEquals("baz", weld.select(Foo.class).get().getBar());
+        assertUnresolvable(weld.select(Alpha.class));
```

Comment:
`WeldInitiator.select()` will return you instance of `WeldInstance`. From there you can make use of `WeldInstance.isResolvable()`- I think that's better than creating a special class just for assertions.